### ISSUE TITLE
Unicorn not starting when using mina

### ISSUE THREAD

**Issue by nmora** [scarfacedeb/mina-unicorn#8]
When I run mina unicorn:start, I get an error:  /home/deployer/xxx-api/releases/7 not found . I am certain that directory exists on the server. Can you kindly take a look to know if it is a problem with mina or mina-unicorn.

```
Navaratans-Mac-mini:xxx-api navtech$ RAILS_ENV=production mina unicorn:start --verbose
-----> Loading environment
$ source ~/.bashrc
-----> Loading rbenv
$ export RBENV_ROOT="$HOME/.rbenv"
$ export PATH="$HOME/.rbenv/bin:$PATH"
$ eval "$(rbenv init -)"
-----> Starting Unicorn...
/home/deployer/xxx-api/releases/7 not found
Connection to 104.131.92.185 closed.

 !     Command failed.
       Failed with status 2560
```

**Comment by gitBasters** [comment]
Yesterday also encountered this problem and and after studying i find next:

mina-unicorn gem have file - utility.rb

This file has a line 
```
cd #{deploy_to}/#{current_path} && BUNDLE_GEMFILE=#{bundle_gemfile} #{unicorn_cmd} -c #{unicorn_config} -E #{unicorn_env} -D;
```

here for some reason is not set variable bundle_gemfile. If I'm not mistaken, it is set in unicorn.rb

```
ENV['BUNDLE_GEMFILE'] = "#{app_dir}/Gemfile"
```

But it is not available in file utility.rb.


I solved this problem in deploy.rb by adding 

```
set :bundle_gemfile,  "#{deploy_to}/#{current_path}/Gemfile"
```


I would like to hear developers comment on this issue

**Comment by tomnatt** [comment]
We encountered exactly the same issue and came to the same conclusion as @gitBasters.

Our deployment scripts have not change in the last few months, however this has just started to be a problem. Is it perhaps linked to the new versions of bundler?

**Comment by rderoldan1** [comment]
+1

**Comment by scarfacedeb** [comment]
Fixed by #9.
I just released 0.1.0 with that fix.
